tests/controls/python_spec.rb

```diff
@@ -0,0 +1,39 @@
+title 'Python & Libraries'
+
+control 'Python version' do
+  impact 'high'
+  title 'Python should be a specific version'
+  desc 'Python should be a specific version'
+  tag 'python'
+  tag 'version'
+
+  describe command('python --version') do
+    its('stdout') { should match /Python 3.7.1/ }
+    its('exit_status') { should eq 0 }
+  end
+end
+
+control 'Pandas can read a CSV' do
+  impact 'high'
+  title 'Python Pandas is installed and can read a CSV'
+  desc 'Python Pandas is installed and can read a CSV'
```

Comment:
You might wish to add a good reason - it is to test that python packages in general install ok, for example because data engineers invite RStudio users to install [s3tools](https://github.com/moj-analytical-services/s3tools) and dbtools, which include python.